content/en/docs/reference/config-api/client-authentication.v1beta1.md

```diff
@@ -0,0 +1,270 @@
+
+
+
+
+
+---
+title: client.authentication.k8s.io/v1beta1
+content_type: tool-reference
+---
+
+
+## Resource Types 
+
+
+  
+- [ExecCredential](#client-authentication-k8s-io-v1beta1-ExecCredential)
+  
+    
+
+
+## `ExecCredential`     {#client-authentication-k8s-io-v1beta1-ExecCredential}
+    
+
+
+
+
+ExecCredential is used by exec-based plugins to communicate credentials to
+HTTP transports.
+
+<table class="table">
+<thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
+<tbody>
+    
+<tr><td><samp>apiVersion</samp><br/>string</td><td><samp>client.authentication.k8s.io/v1beta1</samp></td></tr>
+<tr><td><samp>kind</samp><br/>string</td><td><samp>ExecCredential</samp></td></tr>
+    
+
+  
+  
+<tr><td><samp>spec</samp> <B>[Required]</B><br/>
+<a href="#client-authentication-k8s-io-v1beta1-ExecCredentialSpec"><samp>ExecCredentialSpec</samp></a>
+</td>
+<td>
+   Spec holds information passed to the plugin by the transport.<table>
+
+  
+<tr><td><samp>cluster</samp><br/>
+<a href="#client-authentication-k8s-io-v1beta1-Cluster"><samp>Cluster</samp></a>
```

Comment:
The rendered output is a bit odd or squashed in the table?

Reply:
will check

Reply:
Ah ... I should have treated the `spec` field as other fields, i.e. flatten the layout rather than embedding a table. Will fix.